SuiteCRM/SuiteCRM
# Remove unused EmailImage entrypoint

## Description
We have several entry-points to get images (for Email campaigns, etc) and this one seems old and unused.

We're using others, and there seems to be an unnecessarily high number of them - `download`, `getImage`, `image`...

## How To Test This
I couldn't find any uses of this entry-point.

I found this [old PR](https://github.com/salesagility/SuiteCRM/commit/01e44aabf606ccc8169121bb668376c89c1c18d6#diff-6487b2569c833e951f526dc0c89ba89f) which touches the file, but it seems accidental, the other changes don't use the file. Anyway that PR is a nice place to see which files are involved in attached email images.

## Types of changes


### Final checklist
